ci-operator/config/open-cluster-management/console-api/open-cluster-management-console-api-release-2.4.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
+  base:
+    name: builder
+    namespace: open-cluster-management
+    tag: nodejs14-linux
+  open-cluster-management_builder_nodejs14-linux:
+    name: builder
+    namespace: open-cluster-management
+    tag: nodejs14-linux
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: nodejs14-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  inputs:
+    open-cluster-management_builder_nodejs14-linux:
+      as:
+      - registry.ci.openshift.org/open-cluster-management/builder:nodejs14-linux
+  to: console-api
+promotion:
+  disabled: true
+  name: "2.4"
+  namespace: open-cluster-management
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: pr-image-mirror
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: console-api
+    env:
+      IMAGE_REPO: console-api
+      REGISTRY_ORG: open-cluster-management
+    workflow: ocm-ci-image-mirror
+- as: unit-tests
+  commands: |
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow install
+    make -f Makefile.prow unit-tests
+  container:
+    from: src
+- as: sonarcloud
+  commands: |
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow sonar/js/jest-init
+    make -f Makefile.prow unit-tests
+    make -f Makefile.prow sonar/js/prow
+  container:
+    from: src
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
+- as: sonar-post-submit
+  commands: |
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow sonar/js/jest-init
+    make -f Makefile.prow unit-tests
+    make -f Makefile.prow sonar/js/prow
+  container:
+    from: src
+  postsubmit: true
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
```

Comment:
@ray-harris Following https://github.com/open-cluster-management/cicd-docs/blob/main/prow/new_release.md, there was already a `sonarcloud` test here, but not as a postsubmit, so I added `sonar-post-submit` as well. Does this look right?

Reply:
Yes